### PR TITLE
Postmark API 

### PR DESCRIPTION
**Pull request by passiweinberger**
#166

### CODE REVIEW

File: utils/mailman.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+var postmark = require("postmark");
+var SERVER_KEY = process.env.POSTMARK_API_TOKEN;
+var SENDER = "help@numenta.org";
+
```
Comment (rhyolight) [inline utils/mailman.js]: To keep with the style of the rest of the code:

```
var postmark = require("postmark")
   , SERVER_KEY = process.env.POSTMARK_API_TOKEN
   , SENDER = "help@numenta.org"
   ;
```

Also, what happens if the `POSTMARK_API_TOKEN` isn't set? Probably a "property missing" exception. Better would be to [throw a more detailed error](https://github.com/numenta/nupic.tools/blob/master/utils/config-reader.js#L102) on startup.

File: utils/mailman.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+sing the specified sender mail-adress and the enviornment-variable POSTMARK_API_TOKEN
+*/
+var client = new postmark.Client(SERVER_KEY);
+client.sendEmail({
+    "From": SENDER, 
+    "To": To, 
+    "Subject": Subject, 
+    "TextBody": Body
+});
+return 0
```
Comment (rhyolight) [inline utils/mailman.js]: No need to `return 0` here. You can just leave this out.

File: utils/mailman.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+var postmark = require("postmark");
+var SERVER_KEY = process.env.POSTMARK_API_TOKEN;
+var SENDER = "help@numenta.org";
+
+function sendMail(To, Subject, Body){
```
Comment (rhyolight) [inline utils/mailman.js]: Lower case `to`, `subject`, `body`. Upper case implies `Class` objects.

Comment (rhyolight) [inline utils/mailman.js]: Also, this function should include a `callback` as the last parameter, which is invoked after calling `client.sendEmail()`.

File: utils/mailman.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+A simplified function using the Postmark Email-Client
+sing the specified sender mail-adress and the enviornment-variable POSTMARK_API_TOKEN
+*/
+var client = new postmark.Client(SERVER_KEY);
+client.sendEmail({
+    "From": SENDER, 
+    "To": To, 
+    "Subject": Subject, 
+    "TextBody": Body
+});
```
Comment (rhyolight) [inline utils/mailman.js]: Postmark's `sendEmail` function also accepts a `callback` parameter, see https://github.com/wildbit/postmark.js. You should pass it the callback function that users pass into this function, like this:

```
function sendMail(to, subject, body, callback){
    var client = new postmark.Client(SERVER_KEY);
    client.sendEmail({
        "From": SENDER
      , "To": to
      , "Subject": subject
      , "TextBody": body
    }, callback);
}
```

This way, users of this module can find out if their postmark call was successful.

**Comment by rhyolight** [comment]
Thanks, @passiweinberger. Aside from the inline comments, the indentation seems to be off.

File: utils/mailman.js (hunk 2 above)
Comment (passiweinberger) [inline utils/mailman.js]: Generally, in what cases can you leave the 0 error code out?

**Comment by rhyolight** [comment]
@passiweinberger Build is failing because no `POSTMARK_API_TOKEN` is set in the environment, which is the right behavior, but you'll need to add a dummy `POSTMARK_API_TOKEN` here: https://github.com/numenta/nupic.tools/blob/master/.travis.yml#L9.

**Comment by rhyolight** [comment]
@passiweinberger Thank you! :+1:

**Pull request merged**